From d6f0a63e12ab655e15454d1bbe2f3700202e5056 Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Fri, 5 Nov 2021 14:12:57 -0500
Subject: [PATCH 070/261] docs: Fix broken links in development docs

Use the Git Book as a reference for documentation on Git as no other link
was provided. Other links were broken because they used @url instead of
@uref and needed a comma separator between link and link text.

Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 docs/grub-dev.texi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/docs/grub-dev.texi b/docs/grub-dev.texi
index cd59da66c..c41f38fe2 100644
--- a/docs/grub-dev.texi
+++ b/docs/grub-dev.texi
@@ -96,8 +96,8 @@ This edition documents version @value{VERSION}.
 @node Getting the source code
 @chapter Getting the source code
 
-GRUB is maintained using the @uref{GIT revision
-control system}.  To fetch:
+GRUB is maintained using the @uref{https://git-scm.com/book/en/v2,
+GIT revision control system}.  To fetch:
 
 @example
 git clone git://git.sv.gnu.org/grub.git
@@ -346,8 +346,8 @@ manual and try GRUB 2 out to see what you think is missing from there.
 
 Here are additional pointers:
 @itemize
-@item @url{https://savannah.gnu.org/task/?group=grub GRUB's Task Tracker}
-@item @url{https://savannah.gnu.org/bugs/?group=grub GRUB's Bug Tracker}
+@item @uref{https://savannah.gnu.org/task/?group=grub, GRUB's Task Tracker}
+@item @uref{https://savannah.gnu.org/bugs/?group=grub, GRUB's Bug Tracker}
 @end itemize
 
 If you intended to make changes to GRUB Legacy (<=0.97) those are not accepted
@@ -461,7 +461,7 @@ and the FSF clerks have dealt with your copyright assignment.
 @section When you are approved for write access to project's files
 
 As you might know, GRUB is hosted on
-@url{https://savannah.gnu.org/projects/grub Savannah}, thus the membership
+@uref{https://savannah.gnu.org/projects/grub, Savannah}, thus the membership
 is managed by Savannah. This means that, if you want to be a member of this
 project:
 
-- 
2.36.1

